resolver.go

```diff
@@ -226,10 +236,10 @@ func parseDefault(idx int, xs []ast.Expr) string {
 			// TODO: handle accents
 			return t.Value
 		default:
-			panic("unhandled ast.Text value: " + t.Value)
+			return t.Value
```

Comment:
I'd prefer to keep the panics for the unhandled ast expressions.  The comma case is good to leave as-is.